admin/import/plugins/class-import-aioseo-v4.php

```diff
@@ -231,6 +230,7 @@ protected function detect() {
 	 * @return void
 	 */
 	protected function import() {
+		// This is overriden from the import.js and never run.
 		$aioseo_posts_import_action = YoastSEO()->classes->get( Aioseo_Posts_Importing_Action::class );
```

Comment:
The class here is no longer `use`d so even if it's never run the reference should still be correct to avoid confusion.

Reply:
Not entirely sure you meant that @herregroen but I now added the appropriate `use` to reference that class as well, even though it isn't actively ran.

I also improved that function's docblock so that it can describe better what that function would do if it was ever ran.